_features_versioneye/01_integrate.md

```diff
@@ -0,0 +1,24 @@
+---
+# Page title.
+title: Broad Ecosystem Support
+
+# Image should be served via https.
+image: https://res.cloudinary.com/snyk/image/upload/c_scale,q_auto,w_550/v1513193820/vulndb.png
+
+# Used as the alt text.
+image-caption: "A screenshot of the Snyk website showing test reports alongside each project"
```

Comment:
Please use an image that shows more high severity vulns than the one currently in use. Also make sure your .net and PHP flags are set so the vuln db filters on top show all the ecosystems

Reply:
done